pkg/kubefed2/federate/federate.go

```diff
@@ -124,50 +134,54 @@ func NewCmdFederateResource(cmdOut io.Writer, config util.FedConfig) *cobra.Comm
 	return cmd
 }
 
+type FedResources struct {
```

Comment:
Please rename `FedResources` to clearly indicate it's purpose.

Reply:
Done!